pkg/server/server.go

```diff
@@ -88,13 +87,18 @@ func (s *Server) Run(ctx context.Context) error {
 		go http.ListenAndServe(s.cfg.ProfilerAddress, nil)
 	}
 
-	dir := filepath.Join(".", s.cfg.RootDirectory)
+	var dir string
+	if filepath.IsAbs(s.cfg.RootDirectory) {
+		dir = s.cfg.RootDirectory
+	} else {
+		dir = filepath.Join(".", s.cfg.RootDirectory)
+	}
 	if len(s.cfg.RootDirectory) != 0 {
```

Comment:
Should this be checking `dir` instead?

Reply:
If not, feel free to ignore 😄

Reply:
/shrug

I don't see a semantic difference between the two and was happy with the smaller diff. We default the value anyway so defending against `len() == 0` is on the silly side.